packages/material-ui-lab/src/overrides.d.ts

```diff
@@ -0,0 +1,72 @@
+import type { AlertClassKey, AlertProps } from './Alert';
```

Comment:
These need to be basic `import` for TS 3.2. `import type` is only valid in ^3.8

Reply:
Ah didn't know you guys were still targeting that 👍